lib/withdrawer/client.go

```
@@ -0,0 +1,300 @@
+	for skip := int32(0); ; skip += limit {
+		s := skip
+		resp, err := c.zinc.GetApiVVersionWithdrawal(ctx, "1.0", &zinc.GetApiVVersionWithdrawalParams{
+			Status: &status,
+			Limit:  &limit,
+			Skip:   &s,
+		})
+		if err != nil {
+			return nil, err
+		}
+		content, readErr := io.ReadAll(resp.Body)
+		resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)